phys2bids/viz.py

```diff
@@ -14,9 +14,12 @@
 def plot_trigger(time, trigger, fileprefix, tr, thr, num_timepoints_expected,
                  filename, figsize=FIGSIZE, dpi=SET_DPI):
     """
-    Produces a textfile of the specified extension `ext`,
-    containing the given content `text`.
-
+    Produces a figure with three plots:
+    1. Plots the trigger in blue, a block in orange that indicates the start and end of the time,
+    red line for the threshold used for trigger detection
+    2. Same plot but this showing only the intial trigger
+    3. Same plot but this showing only the intial trigger
+    More info in the howto.rst and doc page
```

Comment:
As for matplotlib, if you want to reference the tutorial, you can add its link in a "See Also" section at the end of the docstrings..